dds_web/database/models.py

```diff
@@ -873,3 +873,21 @@ def __repr__(self):
         """Called by print, creates representation of object"""
 
         return f"<File Version {self.id}>"
+
+
+class MOTD(db.Model):
+    """
+    Data model for keeping track of MOTD (message of the day).
+
+    Primary key:
+    - message
+    """
+
+    # Table setup
+    __tablename__ = "motd"
+    __table_args__ = {"extend_existing": True}
+
+    # Columns
+    id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    message = db.Column(db.Text(4294000000), nullable=False, default=None)
```

Comment:
`4294000000` - why this specific value?

Reply:
this was the only way I found to define a `longtext` type for the message column